test/integration/ggrc/models/test_revision.py

```diff
@@ -638,3 +638,24 @@ def test_changes_revision(self):
     self.assertFalse(revisions[0].is_empty)
     self.assertFalse(revisions[1].is_empty)
     self.assertTrue(revisions[2].is_empty)
+
+  def test_created_by_in_revision(self):
+    """Test revision for Risk contains created_by attr and its content."""
+    # pylint: disable=protected-access
+    with factories.single_commit():
+      user = factories.PersonFactory()
+      risk = factories.RiskFactory(created_by=user)
+
+    revision = all_models.Revision.query.filter(
+        all_models.Revision.resource_id == risk.id,
+        all_models.Revision.resource_type == risk.type,
+    ).first()
+
+    expected_content = {u'context_id': None,
+                        u'href': u'/api/people/{}'.format(user.id),
+                        u'type': user.type,
+                        u'id': user.id,
+                        u'email': user.email}
+
+    self.assertIn("created_by", revision._content)
```

Comment:
We can access revision.content without using protected access. And there is no reason to disable the pylint after that.
```suggestion
    self.assertIn("created_by", revision.content)
```

Reply:
done